approval_polls/forms/choice_form.py

```diff
@@ -0,0 +1,12 @@
+__author__ = 'sparky'
+from django import forms
+from approval_polls.models import Choice
+
+
+class ChoiceForm(forms.ModelForm):
+    class Meta:
+        model = Choice
+        exclude = ('poll', )
+
+    choice_text = forms.CharField(label='Choice',
+                                  widget=forms.TextInput(attrs={'class': 'form-control'}))
```

Comment:
While we do not explicitly use the form to auto generate the choice fields it is being used to hook into Django's built in validation to validate the choices.